Services/GEV/Utils/classes/class.gevUserUtils.php

```diff
@@ -2192,4 +2210,95 @@ public function seeBiproSuperior() {
 		
 		return $this->hasRoleIn($roles);
 	}
+
+	public function wbdCheckForNew() {
+		return $this->hasDoneWBDRegistration() && $this->hasWBDRelevantRole() && $this->userExists() && $this->isActive() && !$this->hasSpecialUserId()
+				&& $this->entryDatePassed() && $this->isWBDBWVIdEmpty() && !$this->hasWBDType(self::WBD_TP_SERVICE);
```

Comment:
Das ist im gevWBDDataConnector schon angepasst. Da werden in Zukunft die Leute gesammelt die in den Benutzerdaten auf "1 - Erstanlage" stehen und danach wird der Check hier ausgeführt.